.eslintrc

```diff
@@ -0,0 +1,7 @@
+---
+"parser": "babel-eslint"
```

Comment:
Imma try and put this into the standard config since that's probably a good idea.